src/views/Header/Header.js

```diff
@@ -211,6 +211,11 @@ class Header extends React.PureComponent {
 
     const { transitionPreset } = this.props;
 
+    let { style } = props;
+    if (options.headerLeftContainerStyle) {
+      style = [style, headerLeftContainerStyle];
```

Comment:
should be `options.headerLeftContainerStyle` here? same for others

Reply:
fixed.